superset-frontend/spec/javascripts/views/CRUD/annotationlayers/AnnotationLayersList_spec.jsx

```diff
@@ -156,7 +156,7 @@ describe('AnnotationLayersList', () => {
   });
 
   it('shows/hides bulk actions when bulk actions is clicked', async () => {
-    const button = wrapper.find(Button).at(0);
+    const button = wrapper.find(Button).at(1);
```

Comment:
@dpgaspar @rusackas FYI: Since we have `can_write` instead of `can_add` and `can_delete` etc,  a button (0) `Bulk Actions` is second, because 1st is `Annotation +`. It's side effect of permission change.

Reply:
@nytai can you help/confirm?

Reply:
i'm confused why this would result in the button ordering change.

Reply:
oh, right. This came up in one of @dpgaspar earlier prs. It's because now the "+ Annotation" button is showing up (used be be based on `can_edit`). 👍

Reply:
Yes, I wanted to double-check - on `master` there is just one button. So this is now sure, that wider permission `can_write` shows now both button.

Reply:
nice! thks